main/include/rotate45.h

```diff
@@ -1,12 +1,26 @@
 #ifndef ROTATE45_H
 #define ROTATE45_H
 #include "algo.h"
+#include <cmath>
 enum Direction45 {CLOCKWISE45,COUNTERCLOCKWISE45};
 class Rotate45: public Algorithm {
 public:
     QImage processImage(const QImage *workingModel);
     friend class Rotate45Builder;
 private:
+    QImage& blur_left_upper_angle(QImage& img,
+                                  int len, int height);
+    QImage& blur_right_upper_angle(QImage& img,
+                                  int len, int height);
+    QImage& blur_left_bottom_angle(QImage& img,
+                                  int len, int height);
+    QImage& blur_right_bottom_angle(QImage& img,
+                                  int len, int height);
+    QImage& blur(QImage& img,
+                 QPoint LeftUpperAngle,
```

Comment:
```suggestion
                 QPoint LeftUpperAngle,
```
первое слово в переменной с маленькой буквы